includes/functions.php

```diff
@@ -4352,3 +4352,21 @@ function dokan_user_update_to_seller( $user, $data ) {
         do_action( 'dokan_new_seller_created', $user_id, $vendor->get_shop_info() );
     }
 }
+
+/**
+ * Get new product creation URL.
+ *
+ * @since DOKAN_SINCE
+ *
+ * @return false|string
+ */
+function dokan_get_new_product_url() {
```

Comment:
would be great if you could move this method under includes/product/functions.php file